### PR TITLE
Update consensus mechanisms developer doc intro

### PR DESCRIPTION
**Pull request by elshigori**
Slight edit to avoid explaining consensus as a voting, which leads to incorrect intuition.

### CODE REVIEW

File: src/content/developers/docs/consensus-mechanisms/index.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ To better understand this page, we recommend you first read our [introduction to
 
 ## What is consensus? {#what-is-consensus}
 
-By consensus, we mean that a general agreement has been reached. Consider a group of five people going to the cinema. If three out of five agree on a film, a consensus is achieved — majority rules.
+By consensus, we mean that a general agreement has been reached. Consider a group of few people going to the cinema. If there are no principal disagreements on a film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.
```
Comment (minimalsm) [inline src/content/developers/docs/consensus-mechanisms/index.md:19]:
```suggestion
By consensus, we mean that a general agreement has been reached. Consider a group of people going to the cinema. If there is not a disagreement on a proposed choice of film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.
```

Comment (minimalsm) [inline src/content/developers/docs/consensus-mechanisms/index.md:19]: Choice of word improvements: 

```suggestion
By consensus, we mean that a general agreement has been reached. Consider a group of people going to the cinema. If there is not a disagreement on a proposed choice of film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.
```

Comment (minimalsm) [reply src/content/developers/docs/consensus-mechanisms/index.md:19]: @elshigori What is this supposed to say: "Participants either get the group on their side, either side with the group."

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @elshigori thanks for the PR 😬❤️

Just made some suggestions to make the sentence clearer and asked for clarification on one of the sentences you added.

File: src/content/developers/docs/consensus-mechanisms/index.md (hunk 1 above)
Comment (elshigori) [reply src/content/developers/docs/consensus-mechanisms/index.md:19]: It's just an outline of options for any consensus to move forward: 
* propagate your position among peers, 
* adjust your position to others, 
* or split.

**Comment by minimalsm** [comment]
Merging, thanks! 

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @elshigori for docs

**Pull request merged**